### PR TITLE
Update wiki and wifi script for BCM4377 Bluetooth

### PR DESCRIPTION
**Pull request by AdityaGarg8**

### CODE REVIEW

**Comment by AdityaGarg8** [comment]
I've allowed maintainers to be able to edit this. If anyone gets time, edit the wiki and also review the script.

**Review by Redecorating — COMMENTED** [review]
This makes sense. I can't test it but if it works for others then LGTM

**Comment by AdityaGarg8** [comment]
Any changes to be done in the wiki?

**Comment by networkException** [comment]
@Redecorating should I fixup my commits with yours?

File: docs/state.md [hunk 1]
```diff
@@ -9,12 +9,12 @@ While Linux is usable on all T2 models, some features are limited due to the lac
 - USB
 - Keyboard
 - Camera
-- Wi-Fi
+- Wi-Fi (requires macOS firmware)
+- Bluetooth (requires macOS firmware)
```
Comment (Redecorating) [inline docs/state.md:13]: only for models with BCM4377 Wifi/Bt chip

**Review by Redecorating — APPROVED** [review]

**Comment by Redecorating** [comment]
> @Redecorating should I fixup my commits with yours?

Dp you mean @AdityaGarg8 ?

**Comment by networkException** [comment]
oh yea, the default profile pictures don't help with distinguishing lol

File: docs/state.md (hunk 1 above)
Comment (networkException) [reply docs/state.md:13]: doesnt wifi require firmware on all systems?

Comment (Redecorating) [reply docs/state.md:13]: yea. It's bluetooth specifically that doesn't need it for most systems

Comment (networkException) [reply docs/state.md:13]: ah I wasn't completely sure what line you commented on

**Comment by AdityaGarg8** [comment]
The script copied the correct firmware files for me.

File: docs/state.md [hunk 2]
```diff
@@ -9,12 +9,12 @@ While Linux is usable on all T2 models, some features are limited due to the lac
...
 - Keyboard
 - Camera
-- Wi-Fi
+- Wi-Fi (requires macOS firmware)
+- Bluetooth (requires macOS firmware only for devices with BCM4377 chip)
 - Touch Bar: There is support for the "Touch Bar Keyboard" device configuration, where only the Function Keys or the Media/Brightness Control Keys can be shown. No other graphics can be shown on the Touchbar (this is what Windows with Bootcamp drivers uses).
 
 ## Partially Working
 
-- Bluetooth: Not working on models with the BRCM4377 WiFi chip (MacBookPro15,4, MacBookPro16,3, MacBookAir9,1).
```
Comment (Redecorating) [inline docs/state.md:17]: Maybe mention the issue where it doesn't work well on these models when 2.4GHz Wi-Fi is used at the same time

Comment (AdityaGarg8) [reply docs/state.md:17]: Was it 2.4Ghz or 5HGhz?

File: docs/guides/wifi-bluetooth.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Introduction
 
-This page is a step by step guide to get wifi working on T2 Macs.
+This page is a step by step guide to get Wi-Fi and Bluetooth working on T2 Macs.
```
[the remark this answers is not in the document]
Comment (AdityaGarg8) [reply docs/guides/wifi-bluetooth.md:3]: I dunno how to do that :(

File: docs/state.md (hunk 2 above)
Comment (Redecorating) [reply docs/state.md:17]: 2.4:

> There is, however, one caveat: WiFi/Bluetooth coexistence isn’t properly configured yet, so you will have poor Bluetooth performance if you are connected to a 2.4GHz WiFi network. We recommend turning off WiFi or using a 5GHz network if you want to use Bluetooth at this time; we will be adding coexistence support in the coming weeks.

https://asahilinux.org/2022/07/july-2022-release/

File: docs/guides/wifi-bluetooth.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Redecorating) [reply docs/guides/wifi-bluetooth.md:3]: `git grep wifi \*.md` and then replace them manually I guess, although there's probably a way to make sed replace while confirming each replacement

**Review by Redecorating — APPROVED** [review]

**Pull request merged**